build.cake

```diff
@@ -248,6 +248,24 @@ Task("Clean")
     CleanDirectories("./**/bin", (fsi)=> !fsi.Path.FullPath.StartsWith("tools"));
 });
 
+Task("provision-net6sdk")
+    .Description("Install NET 6")
+    .Does(async () =>
+    {
+        if(IsRunningOnWindows())
+        {
+            InstallMsiOrExe("https://dotnetcli.azureedge.net/dotnet/Sdk/6.0.100-alpha.1.20562.2/dotnet-sdk-6.0.100-alpha.1.20562.2-win-x64.exe", null);
+            InstallMsiOrExe("https://dl.internalx.com/vsts-devdiv/Xamarin.Android/public/net6/4264138/master/8e097b44df981bb4259845cfe2db9ca2aaaedc91/Microsoft.NET.Workload.Android.11.0.100.255.msi", null);
+            InstallMsiOrExe("https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/main/3174e94a178c41cae0a51fa296e52f711957c14a/543/package/Microsoft.NET.Workload.iOS.14.2.100-ci.main.30.msi", null);
```

Comment:
If you want to update `boots` to latest, it can install `.msi` files:

https://github.com/jonathanpeppers/boots/pull/50